tmtools/helpers.py

```diff
@@ -0,0 +1,77 @@
+"""Helper tools to work with TM-Align."""
+
+from Bio.PDB.Structure import Structure
+import numpy as np
+
+
+def apply_tmalign_transformation(original_structure, tmalign_result):
+    """
+    Transform structure based on TM-Align result.
+
+    This function creates a copy of the input structure and applies the rotation
+    matrix and translation vector from a TM-align result to all atoms in the copied
+    structure.
+
+    Parameters
+    ----------
+    original_structure : Bio.PDB.Structure.Structure
+        The original structure to transform.
+    tmalign_result : object
+        An object containing the TM-align transformation data, with attributes:
+        - u : array_like
+            3x3 rotation matrix.
+        - t : array_like
+            Translation vector with 3 elements.
+
+    Returns
+    -------
+    Bio.PDB.Structure.Structure
+        A new structure object with the TM-align transformation applied. The ID of
+        this structure is the original ID with "_tmalign" appended.
+
+    Raises
+    ------
+    AssertionError
```

Comment:
It's better to avoid using `AssertionError` outside of tests, for two reasons: (a) when run with the `-O` flag, Python will skip any `assert` commands, and (b) usually there are more specific exception types available.

Reply:
Will revert input checks, please refer to comments for reasoning.